README.rst

```diff
@@ -179,6 +182,138 @@ Additionally Supported Parameters
 |                 |                                                                             |
 |                 | Default to **None** meaning above message would not be added to the email.  |
 +-----------------+-----------------------------------------------------------------------------+
+|       type      | Set values of other parameters based on predefined setting                  |
+|                 | ``OPENWISP_NOTIFICATION_TYPES``                                             |
+|                 |                                                                             |
+|                 | Defaults to **None** meaning you need to provide other arguments.           |
++-----------------+-----------------------------------------------------------------------------+
+
+Notification Types
+------------------
+**OpenWISP Notifications** simplifies configuring individual notification by using notification types.
+You can think of notification type as a template for notifications. 
+
+These properties can be configured for each notification type: 
+
++------------------+--------------------------------------------------------------------------------+
+|   **Property**   |                         **Description**                                        |
++------------------+--------------------------------------------------------------------------------+
+|      level       | Sets ``level`` attribute of the notification.                                  |
++------------------+--------------------------------------------------------------------------------+
+|      verb        | Sets ``verb`` attribute of the notification.                                   |
++------------------+--------------------------------------------------------------------------------+
+|      name        | Sets display name of notification type.                                        |
++------------------+--------------------------------------------------------------------------------+
+|   description    | Sets ``description`` attribute of the notification.                            | 
++------------------+--------------------------------------------------------------------------------+
+|  email_subject   | Sets subject of the email notification.                                        |
++------------------+--------------------------------------------------------------------------------+
+| message_template | Path to file having template for description of the notification.              |
++------------------+--------------------------------------------------------------------------------+
+
+.. note::
+    A notification type configuration should contain atleast one of ``description`` or ``message_template``
+    settings. If both of them are present, ``description`` is given preference over ``message_template``. 
```

Comment:
what's the difference between `description` and `message_template`?

Reply:
`description` will be a string with placeholders while `message_template` will be path to template file which will be rendered.